igvm/kvm.py

```diff
@@ -268,7 +268,7 @@ def migrate_live(source, destination, vm, domain):
         (source.dataset_obj['os'], destination.dataset_obj['os'])
     )['flags']
 
-    log.info('Starting migration')
+    log.info('Starting migration of server: {}'.format(vm))
```

Comment:
Why only only for live migration?

Reply:
good idea, its now logging for offline migration as well